blackbox/docs/sql/statements/copy-from.rst

```diff
@@ -130,13 +130,6 @@ Can be used to access buckets on the Amazon AWS S3 Service:
 
     s3://[<accesskey>:<secretkey>@]<bucketname>/<path>
 
-If ``accesskey`` and ``secretkey`` are ommited, CrateDB attempts to load the
```

Comment:
should we change that to also note the fallback to config file parameters?

Reply:
Nope..... these params are only for CREATE REPOSITORY. Copy from/to are once off SQL statements, no credentials are exposed through sys table.

Reply:
ah right :)